_drafts/contribute-to-ansible.markdown

```diff
@@ -9,11 +9,17 @@
     become Ansible contributor
 
 ## Prerequisites
-
-    un ordinateur avec python (2.7 ou 3.5)
-    pipenv
-    un compte github
-    git installé
+Contribute to ansible needs some prerequisites:
+- python 2.7
```

Comment:
Do you need python 2.7 or is it optionnaly ? Do you want to force users to use Python 3.5 to provides code Python 3 compatible ? (I think it's a good practice)